wikicurricula-uy/wikicurricula-boilerplate
# Bug fixed #27 -  Ordering by refrence changes according to subject

Bug Fixed #27 

Changes made - 
The order by refrence does not break when we change the subject 

Steps to test - 
- Go to index.html , open it with live server extension
- In browser, click on the sorting items named as "refrence" , you able to see the refrence based on the x-axis, 
- now , change the subject from the subject dropdown, the refrence would not go at the 0th of X-axis.


Video - 

https://github.com/wikicurricula-uy/wikicurricula-boilerplate/assets/97338915/fbbe1174-61a4-4f52-ab3d-1563fa273224
